phpcs-ruleset.xml

```diff
@@ -12,6 +12,7 @@
     <!-- Minified files don't have to comply with coding standards. -->
     <exclude-pattern>*.min.css</exclude-pattern>
     <exclude-pattern>*.min.js</exclude-pattern>
+    <exclude-pattern>profiles/common/modules/features/multisite_readspeaker/js/ReadSpeaker/</exclude-pattern>
```

Comment:
Exactly. I wanted to load the library automatically in makefile. But we have to be logged to download them. It's a good thing to create a new ticket for this.